schemas/2014-04-01-preview/deploymentTemplate.json

```diff
@@ -122,6 +123,7 @@
                         },
                         {
                             "enum": [
+                                "global",
```

Comment:
Suggest not to have this at the top. Have it at the bottom. This should not be the default first value that people see because it is not valid for the vast majority of our resources.